src/main/java/com/jcabi/ssh/SSHByPassword.java

```diff
@@ -123,7 +89,7 @@ public int exec(final String command, final InputStream stdin,
         randomize = true,
         types = IOException.class
     )
```

Comment:
@kpbird retryonfailure annotation is already declared on the parent class for this method. Also, override annotation is missing